### DISCUSSION TITLE
Currently working on a PR for microtonal accidental playback – should tuning adjustments happen in properties panel or behind the scenes?

### DISCUSSION THREAD

**Discussion by randoguyname** [musescore/MuseScore#15420] [Polls]
I'm a bit concerned that hiding the tuning adjustment from the user might led to confusion, and also that the user may want to know by how much the tuning is being adjusted, but at the same time things such as velocity/dynamics are hidden behind the scenes, so maybe this should be too.

Ideally it should go in the "playback" panel of the accidental, as should dynamics like they were in MU3 (at least in my opinion) but that doesn't seem to fit with the current design (vis-a-vis dynamics).

I'm interested to know what the community thinks about this.

**Comment by BennYapari** [comment]
> things such as velocity/dynamics are hidden behind the scenes, so maybe this should be too.

There are plans to add Automation (like in DAWS) to Musescore 4 eventually, so I say show it to the user!
